app/assets/javascripts/movies.js

```diff
@@ -0,0 +1,19 @@
+$(function() {
+  $('html').on('change', 'select#movie_event_ids.chosen', function(){
```

Comment:
I don't look at your app in production, but `change` event could be fired a lot (focus on select field, use keyboard arrow, on worst, slide with pad...). Maybe a little button or a timer could help ?